AirCasting/Dashboard/DashboardView.swift

```diff
@@ -155,6 +155,7 @@ struct DashboardView: View {
                         }
                     }
                 }
+                .padding(5)
```

Comment:
I think cards got too narrow this way. I realize that you probably did it so that the shadow on the sides would be visible, but you can achieve it also by deleting `.padding(.horizontal)` from ZStack and adding it here instead.